fastlane/Fastfile

```diff
@@ -38,80 +67,149 @@ platform :ios do
 
   end
 
-  # Build and publish the Alpha version to TestFlight
+  # Build and publish version to TestFlight
   lane :release_testflight do
-
     # We increment the build number. If you are adding fastlane to an existing project, check what the current build is in itunes connect. 
```

Comment:
Maybe the second part can be erased now, since we are not adding fastlane to an existing project but creating it with it.